### DISCUSSION TITLE
Files mismatch

### DISCUSSION THREAD

**Discussion by PavanKumarCB** [codinasion-archive/.github-re#120] [General]
I have noticed that in the program for reversing a number using recursion, there is a C code that does the job but it does not use recursion. It should instead be in the folder 'program to print reverse of a number'(without recursion). So, to correct this, will an issue be raised or should i fix it and make a PR instead?

**Comment by brundabharadwaj** [comment]
Hi @PavanKumarCB Thanks for notifying.
Kindly  raise issue for the same .
If you wish to contribute , you can go ahead and  raise PR !
I hope this answers your query
Thanks

**Comment by PavanKumarCB** [comment]
okay. will do that
